src/strategies/DeltaNeutralLp.sol

```diff
@@ -84,25 +100,44 @@ contract DeltaNeutralLp is AccessStrategy {
         borrowPrice = uint256(price);
     }
 
-    /// @notice Get price of WETH in USDC (borrowPrice) from Sushiswap. Has 8 decimals.
+    /**
+     * @notice Get price of WETH in USDC (borrowPrice) from Sushiswap. Has 6 decimals.
+     * @dev return the price of the borrowed token in asset unit.
+     * @dev returned value will be in same decimals as asset.
+     */
     function _sushiPriceOfBorrow() internal view returns (uint256 borrowPrice) {
         address[] memory path = new address[](2);
         path[0] = address(borrow);
         path[1] = address(asset);
 
-        uint256[] memory amounts = router.getAmountsOut({amountIn: 1e18, path: path});
+        uint256[] memory amounts = router.getAmountsOut({amountIn: 10 ** borrow.decimals(), path: path});
         // We multiply to remove the 0.3% fee assessed in getAmountsOut
-        return amounts[1] * 1000 / 997;
+        return amounts[1].mulDivDown(1000, 997);
     }
 
-    /// @notice Convert `borrow` (e.g. WETH) to `asset` (e.g. USDC). Has 6 decimals.
-    function _borrowToAsset(uint256 borrowChainlinkPrice, uint256 amountB) internal pure returns (uint256 assets) {
-        assets = borrowChainlinkPrice.mulWadDown(amountB) / 1e2;
+    /**
+     * @notice Convert `borrow` (e.g. WETH) to `asset` (e.g. USDC). Has 6 decimals.
+     * @dev return value will be in same decimals as asset
+     */
+    function _borrowToAsset(uint256 borrowChainlinkPrice, uint256 amountB) internal view returns (uint256 assets) {
```

Comment:
This is hard to read. Maybe use [`decimalAdjustSign`](https://github.com/AffineLabs/contracts/blob/15a7c41bab1d5cdd15a8d76802e39803f323cd6a/src/strategies/DeltaNeutralLpV3.sol#L69-L72) like in the v3 contract?

See [_borrowToAsset](https://github.com/AffineLabs/contracts/blob/15a7c41bab1d5cdd15a8d76802e39803f323cd6a/src/strategies/DeltaNeutralLpV3.sol#L507)